jest.config.js

```diff
@@ -6,7 +6,5 @@ module.exports = {
 	moduleNameMapper: {
 		"obsidian": "<rootDir>/node_modules/obsidian/obsidian.d.ts"
 	},
-	transformIgnorePatterns: [
-		'node_modules/(?!obsidian/.*)'
-	]
```

Comment:
With this I'm seeing
```
    /obsidian-custom-sort/node_modules/.pnpm/obsidian@0.15.9/node_modules/obsidian/obsidian.d.ts:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import { Extension, StateField } from '@codemirror/state';
                                                                                      ^^^^^^

    SyntaxError: Cannot use import statement outside a module

    > 1 | import {FrontMatterCache, requireApiVersion, TAbstractFile, TFile, TFolder} from 'obsidian';
        | ^
      2 | import {
      3 | 	CustomSortGroup,
      4 | 	CustomSortGroupType,

      at Runtime.createScriptFromCode (node_modules/.pnpm/jest-runtime@28.1.3/node_modules/jest-runtime/build/index.js:1796:14)
      at Object.<anonymous> (src/custom-sort/custom-sort.ts:1:1)
```

Reply:
Sounds good -> 9cd57c00fba7ebe7747ec873e0fae408a8f149e3

To have a more consistent local environment, could you commit your `package-lock.json` or `yarn.lock` file?